templates/dashboard/customer/modal/confirm_promote.html

```diff
@@ -1,23 +1,21 @@
+{% extends "dashboard/base_modal.html" %}
+
 {% load i18n %}
 
-<form action="{% url 'dashboard:customer-promote' pk=customer.pk %}" method="post" novalidate>
-  {% csrf_token %}
-  <div class="modal-content">
-    <div class="row">
-      <div class="col s12">
-        <h5>
-          {% trans "Promote to staff member" context "Dashboard confirmation modal" %}
-        </h5>
-      </div>
-    </div>
-    {% trans "Are you sure?" context "Dashboard confirmation modal" %}
-  </div>
-  <div class="modal-footer">
-    <a href="#!" class="modal-action modal-close btn-flat">
-      {% trans "Cancel" context "Dashboard cancel action" %}
-    </a>
-    <button type="submit" class="modal-action btn-flat">
-      {% trans "Promote" context "Dashboard promote action" %}
-    </button>
-  </div>
-</form>
+{% block form_class %}{% endblock %}
+
+{% block action_url %}
+{% url 'dashboard:customer-promote' pk=customer.pk %}
+{% endblock %}
+
+{% block title %}
+{% trans "Promote to staff member" context "Modal promote to staff member title" %}
+{% endblock %}
+
+{% block content %}
+{% trans "Are you sure?" context "Modal promote to staff member text" %}
```

Comment:
I realize this string was already here but I feel that "Are you sure?" should really restate the thing that's about to happen. In its current state it will also look awkward when localized ("Czy na pewno?" to avoid gendered forms).

Reply:
You are right. We can display instead ```Are you sure you want to promote {{ customer }} to staff member?``` as we keep doing in confirm_delete modal templates. Referring to those templates, I would rename all of them from ```confirm_delete``` to ```confirm_remove``` or even ```remove``` to keep consistent naming of removal actions.